### ISSUE TITLE
Child containment minimum cardinality should be respected during model creation

### ISSUE THREAD

**Issue by finger563** [webgme/webgme#942]
When I create a new object `N` that has children of some type `T` with cardinality `X..Y`, where `X>0`, the new object `N` should have `X` children of type `T`.  Obviously this can get hairy when `T` is abstract with multiple other types inheriting from it, but I believe that's resolvable.

**Comment by kecso** [comment]
Let me try to clarify your scenario:
- you have typeN in your language and it has a containment rule that instructs it to have X..Y children of typeT.
- you want every new N of typeN to have X children of typeT when you create it?

If I understand your scenario correctly, then this can be only done if you have your typeN to not just have the Meta rule of containment, but actually contain instances of typeT. 

When you define your language, you just create your meta constraints, but it is up to the modeler to follow them and WebGME cannot enforce those rules in all scenarios.

**Comment by finger563** [comment]
The only reason I added this is because the default controls for the part browser adhere to those constraints (i.e. it won't let me drag more objects than the cardinality allows).  If it is going to adhere to the maximum cardinality constraints, I figured it should adhere to the minimum cardinality constraints as well.  However, I do agree with your point that that concern is probably best left up to the developer of the visualizer when they handle the node creation events.  (again, the modeler and the meta-modeler are two different types of people; generally in our domains we don't want the modelers looking at or having to worry about the meta at all, so enforcing some amount of correct-by-construction is warranted, as it is easier and faster than running constraint checks after the fact.)  I'll look into that.
